From 271cc0f4f6effe75406e565c2a98d702fe2c5c0e Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Wed, 17 Feb 2016 19:29:31 +0000
Subject: [PATCH 0656/1000] MANTIS 40061 extension: Extend Conveyor-Component
 for extended ConveyorDev.idl

codes.idl
V1.3.4

added BAGGAGE_INCORRECTLY_PLACED, BAGGAGE_TUB_REQUIRED, BAGGAGE_HANDLE_EXTENDED, BAGGAGE_TOO_WIDE



git-svn-id: svn://localhost/SelfServiceCommon/trunk@988 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Cuss/codes.idl | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/SelfServiceCommon/Massai/idl/Cuss/codes.idl b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
index 8597a726..c495b2a5 100644
--- a/SelfServiceCommon/Massai/idl/Cuss/codes.idl
+++ b/SelfServiceCommon/Massai/idl/Cuss/codes.idl
@@ -37,6 +37,11 @@
 //  @li Mantis 0037186: CUSS-Spec: missing definition of DS_TYPES_KEY
 //  @li changed values for DS_TYPES_KEY, DS_TYPES_KEY_DOWN, DS_TYPES_KEY_UP
 //                 
+//  @date 27.01.2016
+//  @li V1.3.4
+//  @li Mantis 0040061: Extend Conveyor-Component for extended ConveyorDev.idl
+//  @li added BAGGAGE_INCORRECTLY_PLACED, BAGGAGE_TUB_REQUIRED, BAGGAGE_HANDLE_EXTENDED, BAGGAGE_TOO_WIDE
+//                 
 //----------------------------------------------------------------------------
 
 #ifndef CODES_IDL
@@ -295,6 +300,10 @@ module statuscodes
     const long BAGGAGE_DELIVERED             = 148; /**< <b>Scope:</b> private + platform  */
     const long BAGGAGE_INTERFERENCE_USER     = 149; /**< <b>Scope:</b> private + platform  */
     const long BAGGAGE_INTRUSION_SAFETY      = 150; /**< <b>Scope:</b> private + platform  */
+    const long BAGGAGE_INCORRECTLY_PLACED    = 151; /**< <b>Scope:</b> private + platform; materna ccm extension */
+    const long BAGGAGE_TUB_REQUIRED          = 152; /**< <b>Scope:</b> private + platform; materna ccm extension */
+    const long BAGGAGE_HANDLE_EXTENDED       = 153; /**< <b>Scope:</b> private + platform; materna ccm extension */
+    const long BAGGAGE_TOO_WIDE              = 154; /**< <b>Scope:</b> private + platform; materna ccm extension */
 
     const long FORMAT_ERROR                = 201; /**< <b>Scope:</b> private + platform (platform only for output classes) */
     const long LENGTH_ERROR                = 202; /**< <b>Scope:</b> private + platform (platform only for output classes) */
-- 
2.41.0.windows.1

